agent/internal.go

```diff
@@ -73,7 +73,8 @@ type resumeChatResponse struct {
 }
 
 type deactivateChatRequest struct {
-	ID string `json:"id"`
+	ID                      string `json:"id"`
+	IgnoreRequesterPresence bool   `json:"ignore_requester_presence,omitempty"`
```

Comment:
what's up with all these omitemptys?

Reply:
to underline that the param is optional, doesn't make any functional differences